src/Compilers/VisualBasic/Portable/Binding/ImportAliasesBinder.vb

```diff
@@ -40,6 +41,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                            TypeOf containingBinder.ContainingBinder.ContainingBinder Is SourceModuleBinder)))
         End Sub
 
+        Public Function GetImportChainData() As ImmutableArray(Of IAliasSymbol)
```

Comment:
@AlekseyTs with the new approach i still needed a way to shuttle information out of these binders (to the core helper in vb's semantic model).  Is this ok with you?  or is there a preferred/different way i should be doing instead?

Reply:
(this is all three of the binders that i needed to touch).